ClinGen/clincoded
# Connects to #883. Gene-centric tab.

**Know issue:**
1. The Ensembl link does not persist if I navigate to other tabs and return to the Gene-centric tab. And refreshing the page will cause the link to show again.
2. The list of synonyms under HGNC in the Gene Resources panel has been left out for this release because I need a bit more time to work on it.

**Steps to test:**
1. Login and use 231744 variant id for testing.
2. Proceed to Gene-centric tab and expect to see ExAC Constraint Scores table being populated with data. Also expect to see various links going to the intended destinations by clicking on them.
3. Expect to see MyVariant and MyGene image icons being displayed on various tabs for attribution.
4. Expect to see the _Gene-specific_ tab now reads _Gene-centric_. And _Functional_ now reads _Experimental_.
